src/util/contribution-types.js

```diff
@@ -3,6 +3,10 @@ const repo = require('../repo')
 
 const defaultTypes = function(repoType) {
   return {
+    a11y: {
+      symbol: '️️️️♿️',
+      description: 'Accessibility',
+    }
```

Comment:
Forgot a comma :)

```suggestion
    },
```

Reply:
I don't have the permission to fix it myself. @phacks can you or someone with write access get this fix commited so we can merge the PR? I've already merged the PR for the website: https://github.com/all-contributors/all-contributors/pull/326

Reply:
I tried fixing this but I don't have permission either, that being said I didn't try to fix that locally.

Reply:
oops 🙈 I corrected this in 485e598, sorry about that!